build.gradle

```diff
@@ -48,13 +48,19 @@ dependencies {
 
   optionalJenkinsPlugins 'org.jenkins-ci.plugins:junit:1.6@jar'
   optionalJenkinsPlugins 'org.jenkins-ci.plugins:cobertura:1.9.6@jar'
+  optionalJenkinsPlugins 'org.jenkins-ci.plugins:jacoco:2.0.1@jar'
+  providedCompile 'org.jacoco:org.jacoco.report:0.7.5.201505241946'
```

Comment:
> Do you mind using jacoco 0.8.1 which is the latest stable

> latest jacoco requires asm 6 for java 9/10 support

Jacoco is on the classpath only so compiler can find its symbols for compilation. Jacoco, at runtime, comes from Jacoco Jenkins plugin. Updating Jacoco, therefore, would be out of this PR's scope and would belong in the [Jacoco plugin](https://wiki.jenkins.io/display/JENKINS/JaCoCo+Plugin). The version 3.0.1 [uses](https://github.com/jenkinsci/jacoco-plugin/blob/jacoco-3.0.1/pom.xml#L101) Jacoco 0.8.0.

Reply:
Sounds good